File: fingerprintauthhelper/src/main/java/com/pro100svitlo/fingerprintAuthHelper/FahListener.java
package com.pro100svitlo.fingerprintAuthHelper;

/**
 * Created by pro100svitlo on 8/26/16.
 */
public interface FahListener {
    void onFingerprintStatus(boolean authSuccessful, int errorType, CharSequence errorMess);
    void onFingerprintListening(boolean listening, long milliseconds);
}

File: fingerprintauthhelper/src/kotlin/java/com/pro100svitlo/fingerprintAuthHelper/FahListener.kt
package com.pro100svitlo.fingerprintAuthHelper

/**
 * Created by pro100svitlo on 8/26/16.
 */
interface FahListener {
    fun onFingerprintStatus(authSuccessful: Boolean, errorType: Int, errorMess: CharSequence)
    fun onFingerprintListening(listening: Boolean, milliseconds: Long)
}
